docs/reference/infrastructure/website.md

```diff
@@ -0,0 +1,14 @@
+# Website
+
```

Comment:
image?

Reply:
This is just the reference on how it's built, so I am afraid screenshots (if that's what you mean) would get outdated very quickly.